### ISSUE TITLE
Tracing tools inside toolbox

### ISSUE THREAD

**Issue by fejfighter** [containers/toolbox#497]
TLDR: I want to run bpftrace and (gtk)sysprof inside a toolbox on toolbox originating processes.

I understand this is a lofty goal listed in the README.md but I wanted to through some weight behind it and aim to gather ideas and if I have some spare cycles have a dig into it.

From what I understand, the toolbox container will require matching kernel headers, and access to the /sys filesystem for Bpftrace. There were issues with polkit for Sysprof (probably more, but that was the brick wall I hit).

I'd like to use both in emacs development which so far I have had to resort to other methods

hoping to get spur discussion on this and if I get some spare cycles I might be able to have a crack

**Comment by fejfighter** [comment]
Long time between posts, but if we accept that running a root container to achieve the outcomes, this is largely solved.

There is a little bit of work to get the gtk sysprof trace recorded and later opened on a wayland system, but it's not impossible like it previously was
